### ISSUE TITLE
Signals compat

### ISSUE THREAD

**Issue by josephcsible** [CyclopsMC/IntegratedDynamics-Compat#6]
We should add compat with the [Signals](https://minecraft.curseforge.com/projects/signals) mod. Here's what I have in mind:

- [x] Writing minecarts' schedules (as a list of strings)
- [x] Writing minecarts' next destination (as an integer index)
- [ ] Forcibly changing signals, like the rail network controller lets you do (they can already be read with an ordinary redstone reader)
- ~~Some way of directing items either into the cart's inventory or the engine's inventory, possibly by letting you change a cart hopper's mode~~

**Comment by rubensworks** [comment]
Haven't used that mod myself yet, but this definitely sounds like an interesting compatibility to add.

**Comment by josephcsible** [comment]
@rubensworks Which aspect readers/writers do you think I should use for this? Should I make a "Block Writer” and "Entity Writer”?

**Comment by rubensworks** [comment]
Perhaps "Machine Writer" instead of "Block Writer" might be better? "Entity Writer" sounds good.

**Comment by josephcsible** [comment]
Blocked by CyclopsMC/IntegratedDynamics#375

**Comment by rubensworks** [comment]
Can't you already start implementing it already using that feature branch of ID? I'd rather not merge it in master until the feature release.

**Comment by josephcsible** [comment]
Yeah, I can. Just noting that it won't be mergeable without that.

**Comment by rubensworks** [comment]
Closing since the mod doesn't seem to exist anymore.
